src/java.base/share/classes/java/net/doc-files/net-properties.html

```diff
@@ -224,6 +224,12 @@ <H2>Misc HTTP URL stream protocol handler properties</H2>
 	      property is defined, then its value will be used as the domain
 	      name.</P>
 	</OL>
+	<LI><P><B>{@systemProperty http.auth.digest.reEnabledAlgs}</B> (default: &lt;none&gt;)<BR>
```

Comment:
Hello Michael, from what I understand of `doc-files` directory (which is where this html file resides) the `javadoc` tool considers it an unprocessed files location. So would adding the `systemProperty` javadoc taglet here be necessary? I think this won't end up being listed in the system properties index generated by the javadoc tool and I think this line here will just get rendered literally.

Reply:
This actually seems to work. If you build the docs for JDK mainline, you can search for instance, for `http.keepAlive.time.proxy` in the javadoc search box and it will lead you `net-properties.html`.

Reply:
You are right indeed. I just checked the generated system properties index and it does list these properties and even links back to this document. In fact, I even found a mail which clearly states that this processing of `@systemProperties` is supported for `doc-files` https://mail.openjdk.java.net/pipermail/core-libs-dev/2018-November/056653.html